project/README.md

```diff
@@ -0,0 +1,126 @@
+<!-- omit from toc -->
+# My Project FilmDrop Deployment
+
+- [Development](#development)
+- [Pre-deploy setup](#pre-deploy-setup)
+- [Deploying](#deploying)
+  - [Manually](#manually)
+  - [Via GitLab CI](#via-gitlab-ci)
+  - [Via GitHub Actions](#via-github-actions)
+- [Destroy](#destroy)
+  - [Manually](#manually-1)
+  - [Via GitLab CI](#via-gitlab-ci-1)
+  - [Via GitHub Actions](#via-github-actions-1)
+
+This repository deploys the
+[FilmDrop Infrastructure Terraform Modules](https://github.com/Element84/filmdrop-aws-tf-modules).
+
+## Development
+
+Install pre-commit hooks:
+
+```bash
+pre-commit install
+```
+
+And run them with:
+
+```bash
+pre-commit run --all-files
+```
+
+## Pre-deploy setup
+
+1. For each of the AWS Accounts to be deployed into, create the bootstrap
+   resources as outlined in <bootstrap/README.md>.
```

Comment:
This file doesn't seem to exist. I worked out that the info was in the deploy.md file in the repo root, but it threw me when I couldn't find bootstrap instructions in the copied project.